connector/protobuf/README.md

```diff
@@ -34,3 +34,17 @@ export SPARK_PROTOC_EXEC_PATH=/path-to-protoc-exe
 
 The user-defined `protoc` binary files can be produced in the user's compilation environment by source code compilation, 
 for compilation steps, please refer to [protobuf](https://github.com/protocolbuffers/protobuf).
+
+### Work with IDE
+
+#### IntelliJ IDEA
+
+If you encounter the problem that the JAVA files generated by protobuf are not indexed by IDEA,
```

Comment:
Hm, you could make it working by explicitly running protobuf generation step via Maven Tools dialog.

Reply:
How? Should I close this PR or modify it for this?

Reply:
Maven Tools view. You can select the step and execute what you need to do.

![Screen Shot 2023-03-15 at 8 49 53 AM](https://user-images.githubusercontent.com/6477701/225167213-e81505a3-e82b-4f11-82d3-edc5aacb5d8c.png)

![Screen Shot 2023-03-15 at 8 51 07 AM](https://user-images.githubusercontent.com/6477701/225167217-9293c6d2-0a45-4f15-a57b-5f48766bf39a.png)

Reply:
hi, it can generate protobuf java file, but the file also can't be indexed if not change idea setting. As @LuciferYang say, `the StoreTypes generated in the core module have exceeded the default limit of IntelliJ`. The `Spark Protobuf` module can't do anything for it.